### DISCUSSION TITLE
aiuto con l'uso delle api

### DISCUSSION THREAD

**Discussion by Sbaccius** [Lioydiano/Classeviva#21] [Domande e Risposte]
Uso da poco python e non riesco a comprendere la documentazione delle api, provo a usare la funzione Utente.accedi ma richiede un argomento (self) che non mi è chiaro cosa sia

**Comment by FLAK-ZOSO** [comment]
Allora, questa è una sintassi di Python, prova a leggere quanto segue...
- https://stackoverflow.com/questions/2709821/what-is-the-purpose-of-the-self-parameter-why-is-it-needed
- https://www.w3schools.com/python/gloss_python_self.asp
In particolare devi prima definire un oggetto Utente...
```py
u = Utente("S1234556789", "mia_password")
```
E soltanto dopo chiamare la funzione accedi implicitamente...
```py
u()
```

Prova a leggere la guida...
- https://classeviva.readthedocs.io/it/latest/intro.html
